File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/postgresql/TextSearchTypes.kt
package dev.typr.foundations.docs.postgresql

import dev.typr.kotlinfoundations.*
import dev.typr.kotlinfoundations.data.*

@Suppress("unused")
class TextSearchTypes {
    //start
    // Text search types are available via PgTypes
    // Note: tsvector and tsquery have specialized handling
    val textType: PgType<String> = PgTypes.text
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/postgresql/TextSearchTypes.scala
package dev.typr.foundationssc.docs.postgresql
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object TextSearchTypes:
  //start
  // Text search types are available via PgTypes
  // Note: tsvector and tsquery have specialized handling
  val textType: PgType[String] = PgTypes.text
  //stop
